broken-site-reporting/README.md

```diff
@@ -27,6 +27,10 @@ Test suite specific fields:
 - `gpcEnabled` - boolean - if GPC is enabled or not (native apps only) - GPC can be disabled by user or by remote config
 - `errorDescriptions` - array of strings - optional field containing descriptions from errors generated by failed page loads
 - `httpErrorCodes` - array of int - optional field containing the HTTP status codes returned by the page load
+- `openerContext` - string - optional field containing the opener context determined from the page
+- `jsPerformance` - string - optional field containing the js performance value measured on the page
+- `locale` - string - optional field containing the locale of the application (e.g. 'en')
+- `userRefreshCount` - string - optional field containing the number of times the user refreshed the current page
```

Comment:
Highlighting that this is a string here rather than an int. Okay if that's intentional.